### DISCUSSION TITLE
Audio Sources

### DISCUSSION THREAD

**Discussion by elbkr** [lavalink-devs/Lavalink#777] [Q&A] [answered]
Hi, I wante dto know where I can find available prefixes like the ytsearch:string one. I searched on the repo files but I couldn't find anything

**Comment by Ryan5453** [comment]
https://github.com/freyacodes/Lavalink/blob/master/IMPLEMENTATION.md#track-searching

**Comment by topi314** [comment, marked as answer]
For a general list of supported sources/formats see https://github.com/Walkyst/lavaplayer-fork#supported-formats


There are also various lavalink plugins for additional sources
See `PLUGINS.md`
